lib/exercism/team_stream_filters.rb

```diff
@@ -72,7 +72,7 @@ def sql
     # rubocop:disable Metrics/MethodLength
     def views_sql
       <<-SQL
-      SELECT 1 AS id, COUNT(views.id) AS total
+      SELECT '#{team_slug}' AS id, COUNT(1) AS total
```

Comment:
Good point. I think we should use the team_id if we can. Do we have it? If not, do we have the entire team at an earlier point so we could make the id available?